File: help/eng_gb/exercicios da ficha_11/Ficha11_008/Ficha11_008.cs
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha11_008 {

    //Main Function
    static void Main() {
        Console.Write("height : " );
        int Height = int.Parse(Console.ReadLine());
        for( int i = 1; i < Height + 1; i = i + 1 ) {
            line(i ) ;
        }
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void line( int n){
        for( int i = 0; i < n; i = i + 1 ) {
            Console.Write("*" );
        }
        Console.Write("\n" );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:22

File: help/eng_gb/exercicios da ficha_11/Ficha11_008/Ficha11_008.c
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//User Defined Functions
void line( int n );

//Main Function
int main() {
    printf("height : ");
    int Height;
    scanf("%d", &Height);
    int i = 0 ;
    for( int i = 1; i < Height + 1; i = i + 1 ) {
        line(i ) ;
    }
    return 0;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
void line( int n ){
    int i = 0 ;
    for( int i = 0; i < n; i = i + 1 ) {
        printf("*");
    }
    printf("\n");
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:22
